wallet-api/emulator/Main.hs

```diff
@@ -0,0 +1,14 @@
+module Main where
+
+import Control.Concurrent.STM (atomically, newTVar)
```

Comment:
Not sure this file has been `stylish-haskell`'d

Reply:
oh it is not, I can do that but it's something I wanted to bring up. I used to like hindent + stylish but then @krisajenkins pointed out that stylish breaks whitespace in imports and pragmas meaning if you add an import all the others have a diff against them. Just a thought.